packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -182,9 +182,13 @@ export default class {
     await this._servers.update(server)
   }
 
+  getXenServer(id) {
```

Comment:
*async* function.

Reply:
We don't need to use an `async` function her because it'll always returns a promise.

Reply:
What I mean is use an async function instead of `.then`.

Reply:
Ok, so, in which case we should use an `async` function and in which case we should use a chained promise ?

In this case it have just a one promise. A chained promise allows we to do the expression in one line and it don't ruin the visibility.

Reply:
Use async functions unless it's necessary to use a promise chain.

Reply:
I'll use it can i just have the reason to know why i should use it?

Reply:
- clearer syntax
- better stacktraces in modern JS engines

Reply:
Thank you for your clarification, i'll not it :+1: